app/src/main/java/org/schabi/newpipe/fragments/MainFragment.java

```diff
@@ -303,8 +302,9 @@ public Fragment getItem(final int position) {
             final Fragment fragment;
             try {
                 fragment = tab.getFragment(context);
-            } catch (final ExtractionException e) {
+            } catch (final Exception e) {
                 ErrorUtil.showUiErrorSnackbar(context, "Getting fragment item", e);
+                // TODO: show an error fragment instead
```

Comment:
Not sure if this is ready ye (PR says ready for review and is no draft so I assume yes)

But this should be addressed before merging ;)

Reply:
Fixed

<img width="355" height="404" alt="image" src="https://github.com/user-attachments/assets/69e02d9e-ea3b-4589-a1bf-f04c21011d0b" />

Reply:
> I think that this can be done later / on the refactor branch. I don't have time for that now, but if you want to implement this, fell free to do so ;)

Code is not an issue tracker, please open a issue for this ;)